retrospective-update/cloud_logger.py

```
@@ -0,0 +1,37 @@
+    webhook_post_url: str
+    summary_attributes: dict = {}
+
+    def __init__(self, webhook_url: str = None) -> None:
+        """
+        Initializes the CloudLog object.
+        """
+        self.webhook_post_url = os.getenv('WEBHOOK_URL', webhook_url)
+        self.start_time = datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+
+        if self.webhook_post_url is None or self.webhook_post_url == '':
+            warnings.warn("No webhook URL provided. No logging will be performed.")
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)